controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1398,7 +1401,18 @@ func (r *BareMetalHostReconciler) doServiceIfNeeded(prov provisioner.Provisioner
 		}
 	}
 
-	hasChanges := fwDirty || hfsDirty
+	if liveFirmwareUpdatesAllowed {
+		var errhfc error
```

Comment:
nit: you could call this `err`, there is not much harm in that

Reply:
Done